### PR TITLE
TLUNIZIN-849 Remove hard coded lti key

### PR DESCRIPTION
**Pull request by dovek**
Due Date: 3/9
@dlhaines 
@pushyamig

### CODE REVIEW

File: src/main/java/edu/umich/ctools/sectionsUtilityTool/SectionsUtilityToolServlet.java [hunk 1]
```diff
@@ -180,28 +181,53 @@
 			"Secondary Instructor",
 			"Faculty Grader",
 			"Graduate Student Instructor"));
-
+
```
Comment (dlhaines) [inline src/main/java/edu/umich/ctools/sectionsUtilityTool/SectionsUtilityToolServlet.java]: White space change

File: src/main/java/edu/umich/ctools/sectionsUtilityTool/SectionsUtilityToolServlet.java [hunk 2]
```diff
@@ -180,28 +181,53 @@
...
+		// the value isn't yet set.
+
+		if (appExtSecurePropertiesFile != null) {
+			M_log.debug("oauthCredentials were injected");
+		}
 
+		//  Try getting the oauth credentials properties file for the URL.  They should
+		// not be in the main properties file they contain secrets.
+		if (appExtSecurePropertiesFile == null) {
+			M_log.debug("try to set oauth credentials from url");
```
Comment (dlhaines) [inline src/main/java/edu/umich/ctools/sectionsUtilityTool/SectionsUtilityToolServlet.java]: clarify "from url"?

File: src/main/java/edu/umich/ctools/sectionsUtilityTool/SectionsUtilityToolServlet.java [hunk 3]
```diff
@@ -180,28 +181,53 @@
...
+		// The oauth properties might have been injected on startup, so only set if
+		// the value isn't yet set.
+
+		if (appExtSecurePropertiesFile != null) {
+			M_log.debug("oauthCredentials were injected");
+		}
 
+		//  Try getting the oauth credentials properties file for the URL.  They should
+		// not be in the main properties file they contain secrets.
+		if (appExtSecurePropertiesFile == null) {
```
Comment (dlhaines) [inline src/main/java/edu/umich/ctools/sectionsUtilityTool/SectionsUtilityToolServlet.java]: Use if / then / else ?

File: src/main/java/edu/umich/ctools/sectionsUtilityTool/SectionsUtilityToolServlet.java [hunk 4]
```diff
@@ -216,6 +242,8 @@ public void fillContext(Context context, HttpServletRequest request) {
 	public void storeContext(Context context, HttpServletRequest request) {
 		Map<String, String> ltiValues = new HashMap<String, String>();
 
+//		resetErrorStrings(context);
```
Comment (pushyamig) [inline src/main/java/edu/umich/ctools/sectionsUtilityTool/SectionsUtilityToolServlet.java]: Remove commented code

File: src/main/java/edu/umich/ctools/sectionsUtilityTool/SectionsUtilityToolServlet.java [hunk 5]
```diff
@@ -180,28 +181,53 @@
...
 
+		//  Try getting the oauth credentials properties file for the URL.  They should
+		// not be in the main properties file they contain secrets.
+		if (appExtSecurePropertiesFile == null) {
+			M_log.debug("try to set oauth credentials from url");
+			appExtSecurePropertiesFile = Utils.loadProperties(CCM_SECURE_PROPERTY_FILE_PATH);
+		}
+
+		// if there there still isn't a properties object then use a default set of properties.
+		if (appExtSecurePropertiesFile == null) {
```
Comment (pushyamig) [inline src/main/java/edu/umich/ctools/sectionsUtilityTool/SectionsUtilityToolServlet.java]: there is repeated twice

File: src/main/java/edu/umich/ctools/sectionsUtilityTool/SectionsUtilityToolServlet.java [hunk 6]
```diff
@@ -180,28 +181,53 @@
...
+		}
 
+		//  Try getting the oauth credentials properties file for the URL.  They should
+		// not be in the main properties file they contain secrets.
+		if (appExtSecurePropertiesFile == null) {
+			M_log.debug("try to set oauth credentials from url");
+			appExtSecurePropertiesFile = Utils.loadProperties(CCM_SECURE_PROPERTY_FILE_PATH);
+		}
+
+		// if there there still isn't a properties object then use a default set of properties.
```
Comment (pushyamig) [inline src/main/java/edu/umich/ctools/sectionsUtilityTool/SectionsUtilityToolServlet.java]: there is repeated twice

**Comment by pushyamig** [comment]
Done. Looks Good!

**Pull request merged**